content/blog/2021-08-17-august-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,260 @@
+﻿---
+title: August'21 Heartbeat
+date: 2021-08-17
+description: |
+  Monthly updates are here! The new data centricity focus, a comparison of DVC, 
+  MLFlow and Metaflow, tutorials and tool stacks, doc updates and more!
+descriptionLong: |
+  This month you will find:
+  - 🧑🏽‍💻 Data-centric for the win,
+  - 🧐 Comparison of DVC, MLFlow and Metaflow,
+  - 🛠 Tutorials and Tool Stacks,
+  - 📈 DVC + Streamlit = ❤️,
+  - 📖 Doc Updates,
+  - 🎥 July Meetup Video available,
+  - 🚀 and more!
+picture: 2021-08-17/august21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/august-heartbeat/843
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - Streamlit
+---
+
+# It's all about that Data!
+
+![Data! Data! Data!](https://media.giphy.com/media/4FQMuOKR6zQRO/giphy.gif)
+
+# From the Community
+
+This month we are seeing the progression of a couple of pieces from the
+[June Heartbeat](https://media.giphy.com/media/62HBhssMOgdJUZQp1X/giphy.gif) as
+well as checking out a use case, tool stack, and some great tutorials of our
+Community members.
+
+## LJ Miranda synthesizes the MLOps space once again!
+
+[LJ Miranda](https://twitter.com/ljvmiranda921) writes another amazing article
+after the series of articles he wrote covering the MLOps tools landscape we
+covered in the June Heartbeat. This time he focuses on the wave of data-centric
+focus taking over the space giving a review of the methods, approaches, and
+techniques to ensure quality data for ML projects. If the adroit summaries of
+complex concepts doesn't thrill you, the links to no less than 63 (😱) resources
+will get you on your way to data-centric nirvana.
+
+![Data Centric Framework](/uploads/images/2021/08-17/lj-miranda-data-centric.png)
```

Comment:
This one

Reply:
Before I merge, let me post here what the actual URL paths are in the rendered page for the record.
For this one it's

/static/6cfa523455e454fb01e9f7fabb1cf96f/fa73e/lj-miranda-data-centric.webp